pkg/daemons/agent/agent.go

```diff
@@ -176,6 +176,8 @@ func checkCgroups() (root string, hasCFS bool, hasPIDs bool) {
 				i := strings.LastIndex(last, ".slice")
 				if i > 0 {
 					root = "/systemd" + last[:i+len(".slice")]
+				} else {
+					root = "/systemd"
```

Comment:
Seems odd that /systemd would fix this.  Does the below also address the problem?

```suggestion
					root = "/"
```

Reply:
i tried it.
it yielded the messages like the following.

> node_1    | W0806 02:51:34.700760       1 container_manager_linux.go:608] [ContainerManager] Failed to ensure state of "/": failed to move PID 1 (in "/docker/2fe79e61aa25ce952427a8261a795af5ecab64fe4dda22af14eb6416f784648e/kube-proxy") to "/": cpuset: cgroup parent path outside cgroup root

the cgroups tree in the container looks like the following.

> spacetanuki% docker-compose exec node find /sys/fs/cgroup -name kube\*
/sys/fs/cgroup/systemd/kube-proxy
/sys/fs/cgroup/systemd/kubepods
/sys/fs/cgroup/pids/kube-proxy
/sys/fs/cgroup/pids/kubepods
/sys/fs/cgroup/hugetlb/kube-proxy
/sys/fs/cgroup/hugetlb/kubepods
/sys/fs/cgroup/net_prio/kube-proxy
/sys/fs/cgroup/net_prio/kubepods
/sys/fs/cgroup/perf_event/kube-proxy
/sys/fs/cgroup/perf_event/kubepods
/sys/fs/cgroup/net_cls/kube-proxy
/sys/fs/cgroup/net_cls/kubepods
/sys/fs/cgroup/freezer/kube-proxy
/sys/fs/cgroup/freezer/kubepods
/sys/fs/cgroup/devices/kube-proxy
/sys/fs/cgroup/devices/kubepods
/sys/fs/cgroup/memory/kube-proxy
/sys/fs/cgroup/memory/kubepods
/sys/fs/cgroup/blkio/kube-proxy
/sys/fs/cgroup/blkio/kubepods
/sys/fs/cgroup/cpuacct/kube-proxy
/sys/fs/cgroup/cpuacct/kubepods
/sys/fs/cgroup/cpu/kube-proxy
/sys/fs/cgroup/cpu/kubepods
/sys/fs/cgroup/cpuset/kube-proxy
/sys/fs/cgroup/cpuset/kubepods
spacetanuki%